docker/docker-compose.yml

```diff
@@ -7,10 +7,13 @@ services:
       dockerfile: docker/Dockerfile
     ports:
       - "3001:3001"
-    command: bash -c "rm -f tmp/pids/server.pid && bin/rake setup && bin/rake about && bin/rails server -b '0.0.0.0' -p 3001"
+    command: bash -c "bin/rake about && bin/rails server -b '0.0.0.0' -p 3001"
```

Comment:
why remove the other commands?  the pid file can be left around (for no good reason) and will need to be removed before the server can start.  Guessing the setup task is to create the DB on first call.

Reply:
These where put back in the latest commit.